App doesn't start and aliases are broken (Alias PR)

The alias PR #62  broke the generator. 

After generating the app nothing starts because of a missing comma on line 36 in `_webpack.dist.config.js`.

The default app is broken too because alias feature isn't working with `./src/styles` and the default App crashes.

It's working if I change it to:

    alias: {
      'styles': '../../../src/styles',
      'components': '../../../src/scripts/components/'
    }

but I hope there is a better way I still haven't figured out.

Error here:

    Module not found: Error: Cannot resolve 'file' or 'directory' /src/styles/normalize.css in /my-path/src/scripts/components
    @ ./src/scripts/components/MyNameApp.js 9:0-31

Awesome generator btw! 
Keep up the good work

--- a/templates/common/_webpack.config.js
+++ b/templates/common/_webpack.config.js
@@ -30,8 +30,8 @@ module.exports = {
   resolve: {
     extensions: ['', '.js'],
     alias: {
-      'styles': './src/styles',
-      'components': './src/scripts/components/'
+      'styles': '../../../src/styles',
+      'components': '../../../src/scripts/components/'
     }
   },
   module: {

--- a/templates/common/_webpack.dist.config.js
+++ b/templates/common/_webpack.dist.config.js
@@ -33,10 +33,10 @@ module.exports = {
   ],
 
   resolve: {
-    extensions: ['', '.js']
+    extensions: ['', '.js'],
     alias: {
-      'styles': './src/styles',
-      'components': './src/scripts/components/'
+      'styles': '../../../src/styles',
+      'components': '../../../src/scripts/components/'
     }
   },
 

